packages/asgardex-binance/src/util.ts

```diff
@@ -28,3 +36,34 @@ export const isTransferFee = (v: Fee | TransferFee | DexFees): v is TransferFee
  * Type guard for `DexFees`
  */
 export const isDexFees = (v: Fee | TransferFee | DexFees): v is DexFees => (v as DexFees)?.dex_fee_fields?.length > 0
+
+/**
+ * Get TxType
+ */
+export const getTxType = (t: BinanceTxType): TxType => {
+  if (t === 'TRANSFER' || t === 'DEPOSIT') return 'transfer'
+  if (t === 'FREEZE_TOKEN') return 'freeze'
+  if (t === 'UN_FREEZE_TOKEN') return 'unfreeze'
+  return 'unkown'
+}
+
+/**
+ * Convert BigSource To BaseAmount
+ */
+export const bigToBaseAmount = (v: BigSource): BaseAmount => {
+  return baseAmount(Big(v).mul(1e8).toString())
+}
+
+/**
+ * Convert BaseAmount To BigSource
+ */
+export const baseAmountToBig = (v: BaseAmount): BigSource => {
+  return v.amount().div(1e8).toString()
+}
+
+/**
+ * Convert BaseAmount To Number
+ */
+export const baseAmountToNumber = (v: BaseAmount): number => {
+  return v.amount().toNumber()
+}
```

Comment:
Please add this helper to `asgardex-util` if you think we will need it all over the place.